roboflow/models/object_detection.py

```diff
@@ -181,21 +213,252 @@ def predict(
             self.api_url += "&image=" + urllib.parse.quote_plus(image_path)
             image_dims = {"width": "0", "height": "0"}
             # POST to the API
-            resp = requests.get(self.api_url)
+            resp = requests.post(self.api_url)
 
         resp.raise_for_status()
         # Return a prediction group if JSON data
         if self.format == "json":
+            resp_json = resp.json()
+
+            if resize:
+                new_preds = []
+                for p in resp_json["predictions"]:
+                    p["x"] = int(
+                        p["x"]
+                        * (
+                            int(original_dimensions[0])
+                            / int(self.preprocessing["resize"]["width"])
+                        )
+                    )
+                    p["y"] = int(
+                        p["y"]
+                        * (
+                            int(original_dimensions[1])
+                            / int(self.preprocessing["resize"]["height"])
+                        )
+                    )
+                    p["width"] = int(
+                        p["width"]
+                        * (
+                            int(original_dimensions[0])
+                            / int(self.preprocessing["resize"]["width"])
+                        )
+                    )
+                    p["height"] = int(
+                        p["height"]
+                        * (
+                            int(original_dimensions[1])
+                            / int(self.preprocessing["resize"]["height"])
+                        )
+                    )
+
+                    new_preds.append(p)
+
+                resp_json["predictions"] = new_preds
+
             return PredictionGroup.create_prediction_group(
-                resp.json(),
+                resp_json,
                 image_path=image_path,
                 prediction_type=OBJECT_DETECTION_MODEL,
                 image_dims=image_dims,
+                colors=self.colors,
             )
         # Returns base64 encoded Data
         elif self.format == "image":
             return resp.content
 
+    def webcam(
```

Comment:
This is cool -- do we have regular video infer built into pip as well?

Reply:
We do not